### PR TITLE
fix: [CDB-80]: move dashboard link to correct position

### PR DESCRIPTION
**Pull request by sean-harness**
##### Summary:

Move the NG Custom Dashboards link back to the correct location in the left side menu.

##### Jira Links:

[CDB-61](https://harness.atlassian.net/browse/CDB-61): Original Jira ticket where the menu item moved under Home
[CDB-80](https://harness.atlassian.net/browse/CDB-80): Bug explaining the location under home is the old location and should be moved back to the main left side bar menu

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by chrisblakely01 — APPROVED** [review]

**Review by willd0yle — DISMISSED** [review]

File: src/modules/10-common/RouteDefinitions.ts [hunk 1]
```diff
@@ -1401,16 +1401,17 @@ const routes = {
   toCEPerspectiveDashboard: withAccountId(() => `/ce/perspective`),
   toCEAnomalyDetection: withAccountId(() => `/ce/anomaly-detection`),
   /********************************************************************************************************************/
-  toCustomDashboard: withAccountId(() => '/home/dashboards'),
+  toOldCustomDashboard: withAccountId(() => '/home/dashboards*'),
```
Comment (abhinavrastogi-harness) [inline src/modules/10-common/RouteDefinitions.ts:1404]: what does `*` mean here?

File: src/modules/45-dashboards/pages/__tests__/DashboardsContext.test.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ describe('DashboardsContext', () => {
 
     const { result } = renderHook(() => useDashboardsContext(), { wrapper })
 
-    const expected = { label: 'common.dashboards', url: '/account/undefined/home/dashboards/folder/shared' }
+    const expected = { label: 'common.dashboards', url: '/account/undefined/dashboards/folder/shared' }
```
Comment (abhinavrastogi-harness) [inline src/modules/45-dashboards/pages/__tests__/DashboardsContext.test.tsx:25]: why is the expected URL containing `undefined`?

File: src/modules/45-dashboards/pages/dashboardView/__tests__/DashboardView.test.tsx [hunk 3]
```diff
@@ -82,7 +82,7 @@ describe('DashboardView', () => {
     renderComponent()
 
     expect(includeBreadcrumbs).toBeCalledWith([
-      { label: 'dashboard name', url: '/account/undefined/home/dashboards/folder/shared/view/undefined' }
+      { label: 'dashboard name', url: '/account/undefined/dashboards/folder/shared/view/undefined' }
```
Comment (abhinavrastogi-harness) [inline src/modules/45-dashboards/pages/dashboardView/__tests__/DashboardView.test.tsx:85]: same

**Review by abhinavrastogi-harness — COMMENTED** [review]
have you tested that the main nav highlighting doesnt break with the new `/home` routing changes?

File: src/modules/10-common/RouteDefinitions.ts (hunk 1 above)
Comment (sean-harness) [reply src/modules/10-common/RouteDefinitions.ts:1404]: `*` is used for a redirect, no-one should be using the `/home/dashboards` path anymore so I setup a redirect to `/dashboards` to ensure no links are broken. I plan to remove it in a future release once I am happy all other links are updated.

**Comment by sean-harness** [comment]
I checked the main nav highlighting and it still works but I noticed that I was still using the `HomeSideNavProps` with the route when I should have been using a minimal layout so I remove that.

**Comment by sean-harness** [comment]
retrigger coverage

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by crystal-fong — APPROVED** [review]

**Pull request merged**